colossus-testkit/src/test/scala/colossus/CallbackMatchersSpec.scala

```diff
@@ -0,0 +1,65 @@
+package colossus
+package testkit
+
+import colossus.service.{UnmappedCallback, Callback}
+import org.scalatest.{MustMatchers, WordSpec}
+
+import scala.util.Try
+
+class CallbackMatchersSpec extends WordSpec with MustMatchers{
+
+  import CallbackMatchers._
+
+  "A CallbackMatcher" should {
+
+    "fail to match if a Callback never executes" in {
```

Comment:
@DanSimon ..I think this is a proper test case for testing when a callback never fires....can you verify?